restapi/src/main/java/io/stargate/web/resources/Converters.java

```diff
@@ -73,15 +72,11 @@
 
 public class Converters {
```

Comment:
Possibly. I'd probably want to first wait for actual use case before making things pluggable, but I agree that this is bit awkward. There are other odd cases that I was wondering about as well: `RequestHandler`, for example, is not at all what name implies -- it is a class with static utility method(s) and not a handler.

Name here is too broad as well and definitely suggests something more (and less) than what it does.

Reply:
Yea I saw that annotation removed from `RequestHandler`.. Lot of mixup here, basically we should avoid static stuff as much as possible, because it makes unit tests way more easier and clearer..

Reply:
Yea, was not sure where that annotation came about. Maybe it was non-static stuff earlier or something.
Definitely true wrt testing especially for code related to request/response handling.